### PR TITLE
Update on Burushaski Urdu Girminas and New Keyboard on Urdu Script

### PR DESCRIPTION
**Pull request by quwatsunny**
Kindly include the two keyboards. 

1.  Burushaski Girminas - updated with a lot of  improvements

2- New Keyboards on Urdu Script based for Burushaski. It was developed on the basis of request made by community members and local writers. 

Kindly include them in the master if possible.

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Burushaski Urdu Phonetic Keyboard Change History
```
Comment (darcywong00) [inline HISTORY.md]: The 3 files at this root folder should be removed
HISTORY.md
LICENSE.md
README.md

**Comment by LornaSIL** [comment]
Let me know when you think it's ready for review.

**Comment by quwatsunny** [comment]
> Let me know when you think it's ready for review.

I think, it is done. Kindly review both the keyboards

**Comment by LornaSIL** [comment]
It looks like we have a merge conflict with the .php file. 

I would like to request that you close this Pull Request and open two new PRs. 

The first PR should JUST be the changes to burushaski_girminas. 
I can see a few immediate issues with those files that need changing. 
* You have inconsistent copyright dates. In some places (LICENSE.md, README.md, kmn, etc) the files say 2018 and in some places (HISTORY.md) it seems clearly 2020. Please make them consistent. If you started the keyboard in 2018 it could be `(c) 2018-2020 Quwat K. Sunny`. If the keyboard was created this year it should be `(c) 2020 Quwat K. Sunny`
* In the .kmn file in the first tab, please select **Features / Add / Desktop On Screen Keyboard**
   * Then go to the On-Screen tab and select "Auto-fill Underlying Layout".
   * Then select "Fill from Layout". This will make the On Screen keyboard display the proper characters.
   * Go to the "build" tab and click on Compile keyboard.
* Next go to the .kps file:
   * In the files tab you should click Add and select readme.htm, welcome_html_2feda655.jpg and the .kvk file from the build folder.
   * In the Details tab you should select the "readme.htm" under Optional Information. Delete 1.0.3 and then select "Package version follows keyboard version".

I think that's about it. Please create a new Pull request and just submit this keyboard. Do not request review from everyone. I will be reviewing this keyboard.

Next, you can submit a separate PR for the burushaski_girminas_urdu keyboard. Again, please do not request a review from everyone. I will try to help you with the first PR and then we can work on the burushaski_girminas_urdu one. It might be best to submit the first one for burushaski_girminas and let's work through all the problems. You may want to make changes to the second keyboard based on what you learn from the first one.

**Comment by quwatsunny** [comment]
it was requested to be closed

**Pull request closed without merging**